### PR TITLE
i3_cheat_sheet: Add alias meta-data and refactor json file

### PR DESCRIPTION
**Pull request by NaveenKarippai**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`i3_cheat_sheet: add alias meta-data and refactor json file`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

The i3wm cheatsheet should be triggered for more queries like
`i3wm cheatsheet`. This could be fixed by adding alias meta-data.
The description of cheatsheet should be updated and json data
should be refactored  to follow docs guidelines.

Provide an overview of the changes this pull request introduces.

* alias meta-data added

* refactored json data according to docs guidelines

* description of cheatsheet updated

###### Which issues (if any) does this fix?

See also: #3175

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@moollaza, @GuiltyDolphin , @zekiel )

---

Instant Answer Page: https://duck.co/ia/view/i3_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @goromlagche

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/i3.json [hunk 1]
```diff
@@ -1,11 +1,17 @@
 {
     "name": "i3 Window Manager",
     "id": "i3_cheat_sheet",
-    "description": "Window manager for Linux",
+    "description": "an improved dynamic, tiling window manager targeting GNU/Linux and BSD OS",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/i3.json]: "Tiling window manager for GNU/Linux and BSD" sounds good to me

Comment (NaveenKarippai) [inline share/goodie/cheat_sheets/json/i3.json]: @moollaza Thanks for the review! updated with new commit object.

**Comment by moollaza** [comment]
@NaveenKarippai this looks good to me! Waiting for @goromlagche to approve and then I will be happy to merge 👍

**Comment by NaveenKarippai** [comment]
@moollaza any update on this?

**Comment by edgesince84** [comment]
Hey @NaveenKarippai - sorry for the wait.  we will sort this out quickly!

**Pull request merged**